admin/tools/docker/base/Dockerfile

```diff
@@ -121,9 +119,9 @@ RUN cd /tmp \
     && rm -rf mysqlproxy
 
 RUN cd /tmp \
-    && git clone https://github.com/lsst/xrootd.git \
+    && git clone https://github.com/xrootd/xrootd.git \
     && cd xrootd \
-    && git checkout ssi-5.3.x \
+    && git checkout master \
```

Comment:
We should use a released tag of XRootD here, and not lastest `master`, otherwise this will float on each build.  Check with Andy H. on which version should be used?

Reply:
Migrated to the stable release tag `v5.6.2`. Verified that this release includes the desired functionality.